.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  - package-ecosystem: gomod
+    directory: "/"
+    schedule:
+      interval: weekly
+      day: monday
```

Comment:
we can also auto assign people to the PRs but I think we should probably handle this with a group, @themantissa what do you think here?

Reply:
A group makes it much less likely to get missed if someone is OOO - perhaps just internal MDBers on your team? I can be in the group as well if you'd like to ensure someone is keeping an eye out?

Reply:
yep, we can make this an internal devs group, I'm no admin here so just let me know the name of the group and I'll add it here